src/Microsoft.Azure.SignalR.Management/DependencyInjectionExtensions.cs

```diff
@@ -124,7 +124,14 @@ public static IServiceCollection AddUserAgent(this IServiceCollection services,
                 throw new ArgumentNullException(nameof(userAgent));
             }
 
-            return services.PostConfigure<ServiceManagerOptions>(o => o.ProductInfo += userAgent);
+            return services.PostConfigure<ServiceManagerOptions>(o =>
+            {
+                if (o.ProductInfo == null)
+                {
+                    throw new InvalidOperationException("Product info is null");
+                }
+                o.ProductInfo += o.ProductInfo + userAgent;
```

Comment:
typo?

Reply:
Which word?

Reply:
o.ProductInfo += o.ProductInfo

Reply:
Got it, I was checking spelling error...